CONTRIBUTING.md

```diff
@@ -229,252 +199,70 @@ Juju uses the `gocheck` testing framework, which is automatically installed
-   (you can't use the `@users.noreply.github.com` email that GitHub provides).
-2. Create an account on [Launchpad](https://launchpad.net/), if you don't
-   already have one.
-3. Ensure the email you used for Git commits is a **verified** email on your
-   Launchpad account. To do this:
-   - Go to your Launchpad homepage (`launchpad.net/~[username]`).
-   - Check the addresses listed under the **Email** heading. If your Git email
-     is listed, you're good.
-   - If not, click "Change email settings".
-   - Add your Git email as a new address.
-   - Follow the instructions to verify your email.
-4. Visit the [CLA website](https://ubuntu.com/legal/contributors), scroll down
-   and press "Sign the contributor agreement".
-5. Read the agreement and fill in your contact details. Ensure that you provide
-   your Launchpad username in the "Launchpad id" box.
-6. Press "I agree" to sign the CLA.
-
-Eventually, your Launchpad account should be added to the
-["Canonical Contributor Agreement" team](https://launchpad.net/~contributor-agreement-canonical).
-You will see it listed under "Memberships" on your Launchpad homepage.
-Once this happens, the CLA check will pass, and we will happily review
-your contribution.
-
-
-Sanity checking PRs and unit tests
-----------------------
-
-All PRs run pre-merge check - unit tests and a small but representative sample
-of functional tests. This check is re-run anytime the PR changes, for example
-when a new commit is added.
-
-You can also initiate this check by commenting `/build` in the PR.
-
-Code review
------------
-
-The Juju project uses peer review of pull requests prior to merging to
-facilitate improvements both in code quality and in design.
-
-Once you have created your pull request, it will be reviewed. Make sure to
-address the feedback. Your request might go through several rounds of feedback
-before the patch is approved or rejected. Once you get an approval from a
-member of the Juju project, you are ready to have your patch merged.
-Congratulations!
-
-Continuous integration
-----------------------
-
-Continuous integration is automated through Jenkins:
-
-The bot runs on all commits during the PRE process, as well as handles merges.
-Use the `/merge` comment to land a PR.
-
-Static Analysis
----------------
-
-Static Analysis can be performed by running `make static-analysis`
-
-Required dependencies for full static analysis are:
- - *nix tools (sh, grep etc.)
- - shellcheck
- - python3
- - go
- - golangci-lint
-
-Documentation
-=============
-
-Juju's documentation is stored in the `doc` directory of the repository. It is
-based on the [Canonical starter
-pack](https://canonical-starter-pack.readthedocs-hosted.com/latest/) and hosted
-on [Read the Docs](https://about.readthedocs.com/).
-
-For general guidance, refer to the [starter pack
-guide](https://canonical-starter-pack.readthedocs-hosted.com/latest/).
+```
+git add .
+git commit -m "docs: new stuff"
```

Comment:
I don't think we should encourage using `git commit -m` since we don't want single line commitments messages.

If you leave out `-m` the terminal will open a text editor, to allow a lengthier message. Admittedly, this might become a little clumsy in this doc to lay that out